src/App.tsx

```diff
@@ -3,10 +3,24 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: ' ',
+  };
+
+  handleChange = (event: any) => {
+    this.setState({
+      query: event.target.value.toLowerCase(),
+    });
+  };
+
+  filterMovies = () => moviesFromServer.filter(e => e.description
+    .includes(this.state.query.toLowerCase())
+    || e.title.toLowerCase().includes(this.state.query.toLowerCase()));
```

Comment:
```suggestion
  filterMovies = () => moviesFromServer.filter(e => (e.description + e.title).toLowerCase()
    .includes(this.state.query.toLowerCase())
   );
```

Reply:
thank you, I added the suggested changes